stock_dashboard_api/templates/email.html

```diff
@@ -0,0 +1,46 @@
+<html>
+<head>
+    <style>
+        * {
+            text-align: center;
+        }
+
+        .head, .footer {
+            background-color: lightblue;
+            color: white;
+        }
+
+        .link {
+            text-decoration: none;
+            color: #2C66E1;
+            padding: 5px;
+            border: 3px solid lightblue;
+            width: fit-content;
+        }
+
+        a {
+            text-decoration: none;
+            color: #2C66E1;
+        }
+    </style>
+</head>
+<body>
+    <div class="head">
+        <h1>Hello {{ recipient }}!</h1>
+    </div>
+    <br>
+    <table width="100%" border="0" cellspacing="0" cellpadding="0">
+        <tr>
+            <td align="center">
+                <h2>You've been invited to view a Stock Dashboard at the following link:</h2>
+            </td>
+            <td align="center">
+                <p class="link">{{ link }}</p>
```

Comment:
But it's not a link :D

Reply:
Gmail makes it a link automatically if it has a link structure, but I will fix this